### PR TITLE
[Doc] PR for Document LDAP support for Platform ( UI )

### PR DESCRIPTION
**Pull request by lizayugabyte**
Addresses https://yugabyte.atlassian.net/browse/PLAT-2824

### CODE REVIEW

File: docs/content/latest/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md
Comment (athulsn) [inline docs/content/latest/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md:36]: We cannot use API description for "LDAP URL" field because we don't have a separate field for port number in the UI.


LDAP URL field will be combination of both "LDAP_URL" & "LDAP_PORT" ( colon separated )
Ex: 0.0.0.0:9000 ( <LDAP_URL>:<LDAP_PORT> )

**Review by athulsn — CHANGES_REQUESTED** [review]

File: docs/content/latest/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md
Comment (lizayugabyte) [reply docs/content/latest/yugabyte-platform/administer-yugabyte-platform/ldap-authentication.md]: Fixed

**Review by athulsn — APPROVED** [review]

**Review by athulsn — CHANGES_REQUESTED** [review]
@lizayugabyte 
Please wait, don't merge. Something new came up, I will be making few more changes to the form.

I will let you know once thats done. You will need to update the doc & screenshot after that.

**Review by TaranSinghania-yb — APPROVED** [review]
Thanks for the quick fix! Approving because the SSL/TLS changes look good! Please do wait for @athulsn's confirmation and updates for final merge (few minor changes are yet to come). Thanks!

**Review by athulsn — APPROVED** [review]
Please update the screenshot that I sent on slack before merging @lizayugabyte

**Pull request merged**
